Adding fWallet and Accrescent

Nick1296 opened the pull request:
Hi! I just found out two interesing apps that I want to add to the list!

One if fWallet, which is a passes / ticket / membership card manager which uses the pkpass format (the format used by Apple Wallet) I created the Wallet category for it but please tell me if you think there is a more appropriate one.

The Other app I am adding is Accrescent, a new F-Droid-like app, with its own repository of applications, so I added it under the Play Store category, since Obtainium is also there.

Let me know if I need to change anything and thanks for curating this list!

Psyhackological commented:
Hi!

Thank you for your contribution. I know Accrescent and I like that addition. fWallet is a new to me, are those applicable in your case?
```text
## AppName
- [x] Installed and tested by me on my main smartphone and used for about 15 minutes or more.
- [x] Has dark theme.
- [x] It has x trackers on [εxodus](εxodus report link).
- [x] It has x F-Droid [Anti-Features](F-Droid / IzzyOnDroid link).
- [x] I use the app on a daily basis and I love almost the entirety of it, so I placed next to it a heart.
```

> thanks for curating this list

Thanks for appreciation!

Nick1296 commented:
Sorry I did not see the list of of things to check, I'll work on that and thanks for the comments! I'll thake care of those too

Nick1296 commented:
Here is the list, I am in the process of adding descriptions!

## fWallet

- [x] Installed and tested by me on my main smartphone and used for about 15 minutes or more.
- [x] Has dark theme.
- [x] It has 0 trackers on [εxodus](https://reports.exodus-privacy.eu.org/en/reports/business.braid.f_wallet/latest/).
- [x] It has 0 F-Droid [Anti-Features](https://f-droid.org/en/packages/business.braid.f_wallet/).
- [ ] I use the app on a daily basis and I love almost the entirety of it, so I placed next to it a heart.

## Accrescent

- [x] Installed and tested by me on my main smartphone and used for about 15 minutes or more.
- [x] Has dark theme.
- [ ] App was not scanned by exodus 
- [ ] App is not on F-Droid
- [ ] I use the app on a daily basis and I love almost the entirety of it, so I placed next to it a heart.

Nick1296 commented:
FIxed icons and added the descriptions, can I ask you to check the fwallet icon? github gives a weird link...

Also can you add a placeholder that represents the app description in CONTRIBUTING.md? 

Thanks again!

Trived7 commented:
Include links to the app next time if possible bro, thank you

Psyhackological commented:
FIxed icons and added the descriptions, can I ask you to check the fwallet icon? github gives a weird link...

Also can you add a placeholder that represents the app description in CONTRIBUTING.md?

Thanks again!

Psyhackological commented:
@Trived7 
> Include links to the app next time if possible bro, thank you

I don't know what you mean?
